packages/styles/scss/components/toggle/_toggle.scss

```diff
@@ -198,4 +198,25 @@
   .#{$prefix}--toggle:active .#{$prefix}--toggle__switch {
     @include high-contrast-mode('focus');
   }
+
+  // Skeleton state
+  .#{$prefix}--skeleton {
+    display: flex;
+    align-items: center;
+  }
```

Comment:
this needs to be a more specific class, looks like it's interfering with DataTable and potentially other skeletons
<img width="1257" alt="image" src="https://github.com/carbon-design-system/carbon/assets/40550942/74030d48-8674-4d98-86c1-0c4313d89baa">

Reply:
Fixed issue with conflicting classnames by renaming the new classes